macros/WCTE.mac

```diff
@@ -142,8 +152,36 @@
 #/gps/ang/minphi 0 deg
 #/gps/ang/maxphi 360 deg
 
-##### NEW
-/Tracking/fractionOpticalPhotonsToDraw 0.0
+# Or you can use the AmBe Calibration Source Generator.
+# This launches two primary partircles: one neutron and one gamma.
+# The energy of the gamma can be:
+# - 4.4 MeV 65% of the time.
+# - 7.7 MeV 8%  of the time.
+# - 0.0 MeV 26% of the time.
+# Neutron Enegy is taken from a simulated realistic AmBe Spectrum depending on the Gamma Energy.
+/mygen/generator ambeevt
+
+# If you want to use the AmBe generator with the BGO placed in the geometry, you need to activate
+# the BGOEvent option (i.e. set it to true).
+# Please note that you cannot have BGOEvent set to true and no BGO Placed, this will raise an error.
+/mygen/BGOEvent true
```

Comment:
why you need this? you just need to check whether BGO geometry exists when using ambeevt generator

Reply:
I coded it so you need to set `/mygen/BGOEvent true` if you want to place the BGO in the geometry (i.e. if you set `/WCSim/BGOPlacement true`). And this only applies if you are using `/mygen/generator ambeevt`.

But I also added the option to use `ambeevt` with no BGO (I.e.` /mygen/BGOEvent false && /WCSim/BGOPlacement false`). 

If you use `/mygen/BGOEvent` with any other generator, it will have no effect, just like if you try to place the BGO with any other generator.

So I think the logic is correct, but I can be confused, of course.

Reply:
do we need the option of `ambeevt` generator without BGO? I thought we only want to make sure BGO exists when we use `ambeevt`.
Unless there are actual needs of `ambeevt` generator without BGO, I prefer to remove such feature to avoid confusion. The `/mygen/BGOEvent` command is just confusing for me

Reply:
You are right Ka Ming, there is no actual need for this. I'll remove it.